packages/oas/README.md

```diff
@@ -119,8 +119,8 @@ Because this library has full TypeScript types and docblocks this README is not
 <!-- prettier-ignore-start -->
 | Method | Description |
 | :--- | :--- |
-| `.getExtension()` | Retrieve a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) if it exists at the root of the API definition.  |
-| `.hasExtension()` | Determine if a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) exists on the root of the API definition. |
+| `.getExtension()` | Retrieve a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specification-extensions) if it exists at the root of the API definition.  |
```

Comment:
all of this work looks crazy tedious and i'd love to figure out how to auto-generate docs like these at some point, but in the meantime... bless u for this